### PR TITLE
Add support for Symfony 6.4

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
| --------------- | -----
| Bug fix?        | yes (cause we do not exclude it on composer)
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

File: src/Bundle/Controller/ContainerAwareTrait.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ResourceBundle\Controller;
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * @internal
+ */
+trait ContainerAwareTrait
```
Comment (loic425) [inline src/Bundle/Controller/ContainerAwareTrait.php]: I've copied the Symfony trait cause it's now deprecated

File: composer.json [hunk 2]
```diff
@@ -71,6 +71,7 @@
         "sylius/grid-bundle": "^1.7 || v1.12.0-ALPHA.1",
         "symfony/dependency-injection": "^5.4 || ^6.0",
         "symfony/dotenv": "^5.4 || ^6.0",
+        "symfony/stopwatch": "^5.4 || ^6.0",
```
Comment (loic425) [inline composer.json:74]: I've added this cause there is a bug with workflow without stopwatch.
These 2 ones are on dev requirements

File: src/Bundle/Controller/ContainerAwareTrait.php [hunk 3]
```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
```
Comment (GSadee) [inline src/Bundle/Controller/ContainerAwareTrait.php]:
```suggestion
 * (c) Sylius Sp. z o.o.
```

File: src/Bundle/Controller/ContainerAwareTrait.php (hunk 1 above)
Comment (GSadee) [reply src/Bundle/Controller/ContainerAwareTrait.php]: It is deprecated but not removed in 6.4, so do we really need to copy it in this version? 🤔

File: tests/Application/config/db.sql
Comment (GSadee) [inline tests/Application/config/db.sql:1]: Is this file added intentionally? Because I don't see the connection with this PR 🤔

Comment (loic425) [reply tests/Application/config/db.sql:1]: oups...

File: src/Bundle/Controller/ContainerAwareTrait.php (hunk 1 above)
Comment (loic425) [reply src/Bundle/Controller/ContainerAwareTrait.php]: Yeah but it's removed on 7.0... But right, maybe adding it in 1.11 (the current development one which contains the new resource metadata system).

File: src/Bundle/Controller/ContainerAwareTrait.php (hunk 3 above)
Comment (loic425) [reply src/Bundle/Controller/ContainerAwareTrait.php]: We did this on the 1.11 branch, but not on the 1.10 (for bugfix)

**Review by diimpp — APPROVED** [review]

**Comment by diimpp** [comment]
@loic425 1.10 is not supported anymore, it should target 1.11

image

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, Loïc! :tada:

**Pull request merged**